setup-elytron-guides/setup_elytron_linux.adoc

```diff
@@ -0,0 +1,148 @@
+    +
+    [source,bash]
+    ----
+    export JAVA_HOME=$(readlink -f /usr/bin/java | sed "s:bin/java::")
+    export PATH=$JAVA_HOME/bin:$PATH
+    ----
+
+    .. Run `source <config file>`, substituting in your config file, to load the changes into the terminal.
+    .. You can undo these changes later on by following the steps in the *Temporary Configuration* section.
+. Now run `java --version` again. You should now successfully see the Java 11 version that was installed.
+
+== Installing Maven
+
+. First, let's check if you already have Maven installed.
+    .. Open a terminal, and run `mvn -v`.
+    .. If the output shows a version number, Maven is already installed and you can jump to the *Installing Git* section. If you get `command not found: mvn` or a similar message indicating you do not have maven installed, then continue with the following steps.
+. Either install Maven with your package manager, or go to https://maven.apache.org/download.cgi[this page] and download the latest Binary zip archive (e.g, `apache-maven-3.8.6-bin.zip`).
+. Extract distribution archive in any directory:
++
+[source,bash]
+----
+unzip apache-maven-3.8.6-bin.zip 
+----
+
+. Configure `PATH` environment variable for Maven:
+    .. Open the configuration file for your shell, and add the following line:
+    +
+    [source,bash]
+    ----
+    export PATH=<path to Maven folder>/apache-maven-3.8.6/bin:$PATH
+    ----
+    .. Run `source <config file>`, substituting in your config file, to load the changes into the terminal.
+    .. You can undo these changes later on by following the steps in the *Temporary Configuration* section.
+. Now run `mvn -v` again. You should now successfully see the Maven version that was installed.
+
+== Installing Git
+
+. First, let's check if you already have Git installed.
+    .. Open a terminal, and run `git --version`.
+    .. If the output shows a version number, Git is already installed and you can jump to the *Cloning WildFly Elytron* section. If you get `command not found: git` or a similar message indicating you do not have Git installed, then continue with the following steps.
+. It is easiest to install Git on Linux using the preferred package manager of your Linux distribution. Here are some examples of how to install Git on common Linux distributions:
+    .. Ubuntu: `sudo apt-get install git`
+    .. Fedora: `sudo dnf install git`
+    .. Arch: `sudo pacman -S git`
+. Now run `git --version` again. You should now successfully see the Git version that was installed.
+. If you already had any IDEs open, close and reopen them now to load your changes.
+
+== Cloning WildFly Elytron
+
+. First, you'll need a GitHub account. If you don't already have one, sign up for one at https://github.com/.
+. Fork the https://github.com/wildfly-security/wildfly-elytron[WildFly Elytron project] into your GitHub account by clicking on the "Fork" button at the top right.
```

Comment:
@cam-rod Another thing I realized is that they should configure their identity otherwise this gets thrown:

```
Author identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.
```

We might want to advise them to configure their identity as well

Reply:
added